dashboard/serializers.py

```diff
@@ -1,28 +1,27 @@
 """
 Provides functionality for serializing a ProgramEnrollment for the ES index
 """
-from decimal import Decimal
-
-from dashboard.api_edx_cache import CachedEdxUserData
+from dashboard.api_edx_cache import CachedEdxUserSearchData
 from dashboard.utils import MMTrack
-from roles.models import (
-    NON_LEARNERS,
-    Role
-)
+from roles.api import is_learner
 
 
 class UserProgramSearchSerializer:
     """
     Provides functions for serializing a ProgramEnrollment for the ES index
     """
-    @staticmethod
-    def calculate_final_grade_average(mmtrack):
```

Comment:
like below, proper separation of concerns. the job of this class is to serialize. one of the jobs of MMTrack is to provide information about final grades. it's just a better place for this method. if the calculation of final grades changes (and, as we know, it will be changing soon), you have one class to fix and you don't need to remember to make the change here